doc/api/stream.md

```diff
@@ -572,6 +572,12 @@ Examples of writable streams include:
 * [TCP sockets][]
 * [child process stdin][]
 * [`process.stdout`][], [`process.stderr`][]
+ 
+#### Event: 'close'
+
+Emitted when the stream and any of its underlying resources (a file descriptor, for example) have been closed. The event indicates that no more events will be emitted, and no further computation will occur.
```

Comment:
I hesitate to move the 'Event: 'close'' block to after the 'Event: 'end'' block because these events seem to be in alphabetical order for better user experience.

The difference between 'close' and 'end' events:

1. 'close' event - Whatever it is tied to, for example a stream, it is done now. This event comes from the `fs.close()` method and is optional. Not all streams emit this event.

2. 'end' event - Waits for the current data to finish buffering. means no more data will be emitted. This event is tied to the `end()` function on Streams.